doc/reference/routers.md

```diff
@@ -0,0 +1,174 @@
+# Routers
+
+- [Routers](#routers)
+  - [Order](#order)
+  - [Art](#art)
+
+Router determines how requests are routed to the corresponding Pipeline for subsequent processing. We currently support two routing policies, `Order` and `Art`, and you can choose a Router that suits your needs based on its characteristics, and we also provide the ability to customize Router, if the built-in Router does not meet your needs, you can choose Write a custom Router.
```

Comment:
`policy` and `strategy` are both used in this doc for the same purpose. I think `strategy` is better, but please double confirm.